whatrecord/bin/graph.py

```diff
@@ -0,0 +1,220 @@
+"""
+"whatrecord graph" is used to parse and generate a relationship graph of
+whatrecord-supported file types.
+"""
+
+import argparse
+import logging
+import re
+from typing import Dict, List, Optional
+
+# from ..access_security import AccessSecurityConfig
```

Comment:
future plans, perhaps?

Reply:
Yeah, technically all could be used for graphing in _some way_ but I'm not sure what it'd look like.

I think at least state notation language would benefit from a state transition diagram one day.

Reply:
Gotcha. It all stood out to me because I usually don't see tons of commented lines in your code.